manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -150,6 +151,28 @@ public function getExtensionConfig($extensionName, array $extensionConfigs, Plug
             return $extensionConfigs;
         }
 
+        $extensionConfigs = $this->setDatabaseServerVersion($extensionConfigs, $container);
+
+        return $this->setPdoDriverOptions($extensionConfigs, $container);
+    }
+
+    /**
+     * Sets path to enable autoloading of legacy Contao modules.
+     *
+     * @param string $modulePath
+     */
+    public static function autoloadModules($modulePath)
```

Comment:
Why did you add this method?

Reply:
It was already there, see lines 190 on the diff.

Reply:
It's already there:

https://github.com/contao/contao/blob/8b07746dea9eecb9527a34fe05cb8ff824cd7aa2/manager-bundle/src/ContaoManager/Plugin.php#L189-L197

Reply:
I originally inserted the new private methods before `public static function autoloadModules` - however, as per the code style rules, `public` methode come before `private` ones, therefore the CS fixer moved it around, resulting in this slightly more unreadable diff ;)